Jenkinsfile.content

```diff
@@ -7,8 +7,22 @@ node('vetsgov-general-purpose') {
     ref = getLatestGitRef("vets-website", branch: "master")
   }
 
-  refStatusState = getGithubCommitState("vets-website", ref)
-  if ("${refStatusState}" != "SUCCESS") {
+  def jobFailed = false
+  def refStatusState
+  while({
+    echo "checking build status..."
+    refStatusState = getGithubCommitState("vets-website", ref)
+    if ("${refStatusState}" == "SUCCESS") {
+      return false
+    } else if ("${refStatusState}" == "PENDING") {
+      sleep 60
```

Comment:
What would happen if we reduced this polling interval?

Reply:
We can drop it down. The only ramification is more API calls to github.